web/src/beta/features/Editor/Map/InspectorPanel/LayerInspector/DataSource/index.tsx

```diff
@@ -1,19 +1,37 @@
+import {
+  LayerConfigUpdateProps,
+  LayerNameUpdateProps
+} from "@reearth/beta/features/Editor/hooks/useLayers";
 import { Collapse } from "@reearth/beta/lib/reearth-ui";
-import { InputField, TextareaField } from "@reearth/beta/ui/fields";
+import { InputField } from "@reearth/beta/ui/fields";
 import { NLSLayer } from "@reearth/services/api/layersApi/utils";
 import { useT } from "@reearth/services/i18n";
 import { styled, useTheme } from "@reearth/services/theme";
-import { FC, useCallback, useMemo, useState } from "react";
+import { FC, useCallback, useEffect, useMemo, useState } from "react";
 
+import ResourceUrl from "./ResourceUrl";
 import CustomPropertiesSchema from "./SketchCustomProperties";
 
 type Props = {
   selectedLayer: NLSLayer;
+  onLayerNameUpdate?: (inp: LayerNameUpdateProps) => void;
+  onLayerConfigUpdate?: (inp: LayerConfigUpdateProps) => void;
 };
 
-const DataSource: FC<Props> = ({ selectedLayer }) => {
+const DataSource: FC<Props> = ({
+  selectedLayer,
+  onLayerNameUpdate,
+  onLayerConfigUpdate
+}) => {
   const t = useT();
   const theme = useTheme();
+  const [localTitle, setLocalTitle] = useState(selectedLayer.title);
+  const [localUrl, setLocalUrl] = useState(selectedLayer.config?.data?.url);
+
+  useEffect(() => {
+    setLocalTitle(selectedLayer.title);
+    setLocalUrl(selectedLayer.config?.data?.url);
+  }, [selectedLayer.config?.data?.url, selectedLayer.title]);
```

Comment:
Maybe better separate them, change one of them should not effect another by theory.